public/static/js/settings.js

```diff
@@ -77,14 +78,15 @@ mciModule.controller('SettingsCtrl', ['$scope', '$http', '$window', 'notificatio
     return formattedFeedback;
   }
 
-  $scope.updateUserSettings = function (new_tz, use_spruce_options, spruce_feedback) {
+  $scope.updateUserSettings = function (new_tz, new_region, use_spruce_options, spruce_feedback) {
```

Comment:
It shouldn't be necessary to take new_region as a parameter - $scope.new_region should be set to this already based on what you added to the html file